### PR TITLE
feat: #111 rp adapter: define jsonld context for issuer response VP

### LINKED ISSUE #111: rp adapter: define jsonld context for issuer response VP
part of #60

### PR DESCRIPTION
**Pull request by llorllale**
closes #111 

This PR:

* defines the CreditCardStatementCredential as follows:
```json
      "CreditCardStatementCredential": {
        "@id": "ex:DIDDocumentCredential",
        "@context": {
          "@version": 1.1,
          "id": "@id",
          "type": "@type",
          "ex": "https://example.org/examples#",
          "credentialSubject": {
            "@id": "ex:credentialSubject",
            "@context": {
              "@version": 1.1,
              "@protected": true,
              "stmt": "schema:Invoice"
            }
          }
        }
      }
```
* imports the `schema.org` and verifiable credential json-ld contexts to speed up tests and minimize remote dependencies (pending: inject the cached jsonld document loaders into the production code)

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: pkg/vc/rp/models.go [hunk 1]
```diff
@@ -24,3 +30,43 @@ type DIDDocCredentialSubject struct {
...
+}
+
+// CreditCardStatementCredentialSubject is the custom credentialSubject of the CreditCardStatementCredential.
+type CreditCardStatementCredentialSubject struct {
+	ID        string              `json:"id"`
+	Statement CreditCardStatement `json:"stmt"`
+}
+
+// CreditCardStatement is a credit card statement.
+type CreditCardStatement struct {
```
Comment (rolsonquadras) [inline pkg/vc/rp/models.go:54]: Why are we defining domain specific data model here ? How's this gonna scale ? eg, bankAccount, creditScore etc.  This doesn't look right.

Comment (llorllale) [reply pkg/vc/rp/models.go:54]: Suggestions are welcome.

Comment (llorllale) [reply pkg/vc/rp/models.go:54]: How can we validate the data contained withing the VCs? Or are we satisfied with the mere presence of a VC that the sender claims satisfies the presentation-definitions ?

Comment (llorllale) [reply pkg/vc/rp/models.go]: @rolsonquadras removed.

I've opened #120 as followup to our discussion this morning.

File: pkg/restapi/rp/operation/operations.go [hunk 2]
```diff
@@ -968,52 +970,44 @@ func (o *Operation) handlePresentationMsg(msg service.DIDCommMsg) error {
...
 
-	return verifiable.ParsePresentation(vpBytes)
-}
-
-func getAttachmentByID(id string, attachments []decorator.Attachment) *decorator.Attachment {
-	for i := range attachments {
-		if attachments[i].ID == id {
-			return &attachments[i]
-		}
+	if !adapterutil.StringsContains(rpvc.CreditCardStatementCredentialType, vc.Types) {
```
Comment (rolsonquadras) [inline pkg/restapi/rp/operation/operations.go:1003]: Here, we still have domain specific `CreditCardStatement` hardcoded in main codebase.

Comment (llorllale) [reply pkg/restapi/rp/operation/operations.go]: @rolsonquadras removed

**Pull request merged**